randovania/games/samus_returns/logic_database/Surface - West.txt

```diff
@@ -168,30 +186,29 @@ Extra - asset_id: collision_camera_017
   > Energy Recharge Station
       Any of the following:
           Spider Boost (Intermediate) and Can Spider Boost
-          Space Jump and Movement (Beginner)
           All of the following:
-              Lightning Armor
+              Lightning Armor or Space Jump
               Baby Metroid or Morph Ball
   > Save Station
       All of the following:
           Baby Metroid
-          Wall Jump (Beginner) or Climb Rooms Vertically (High Jump)
+          Any of the following:
+              Single-wall Wall Jump (Beginner) or Wall Jump (Beginner) or Climb Rooms Vertically (High Jump)
+              Morph Ball and Super Jump (Intermediate) and Unmorph Extend (Intermediate)
+              # Damage Boost off the Gullug
+              Morph Ball and Mid-Air Morph (Intermediate) and Damage Boost (Advanced) and Normal Damage ≥ 20
   > Pickup (Missile Tank)
```

Comment:
Should `Space Jump and Mid-Air Morph (Beginner) and Spider` also be added to this node or nah?

Reply:
For `Below Ceiling Pickup` -> `Pickup (Missile Tank)`? I'd say no, if we already have the requirement in the other connection